architecture-baseline.json

```diff
@@ -0,0 +1,9 @@
+[
+    {
+        "fileName": "/vendor/spryker-eco/braintree/src/SprykerEco/Yves/Braintree/Dependency/Client/BraintreeToShipmentClientInterface.php",
+        "description": "\n      Bridges: The bridge interface has incorrect method 'getAvailableMethods' signature. That violates the rule \"All bridge methods must have exactly the same signature as their interface\"\n    ",
```

Comment:
This is actually not a violation as per my RFC and open ticket to fix the arch sniffer
We really should fix that, bridges should be able to fix up their signature here or as in this case use correct one without the sniffer reporting falsely an error. @serhiipalkin @stereomon @asmarovydlo